samza-rest/src/test/java/org/apache/samza/monitor/TestLocalStoreMonitor.java

```diff
@@ -97,7 +97,7 @@ public void cleanUp()  {
     }
   }
 
-  @Test
+//  @Test
```

Comment:
If there is JIRA for follow-up , can you please add a TODO here with the JIRA number?